source/common/http/BUILD

```diff
@@ -510,7 +510,7 @@ envoy_cc_library(
     srcs = ["header_utility.cc"],
     hdrs = ["header_utility.h"],
     external_deps = [
-        "nghttp2",
+        "quiche_http2_adapter",
```

Comment:
I'm by no means a bazel expert, but should this continue to reference nghttp2 until the runtime flag is removed?